Source/Core/DolphinWX/Cheats/GeckoCodeDiag.cpp

```diff
@@ -75,12 +75,16 @@ CodeConfigPanel::CodeConfigPanel(wxWindow* const parent) : wxPanel(parent)
   SetSizerAndFit(sizer_main);
 }
 
+bool helpOrderCodes(GeckoCode& code1, GeckoCode& code2) { // to sort the codes by name
+  return (code1.name < code2.name);
+}
+
 void CodeConfigPanel::UpdateCodeList(bool checkRunning)
 {
   // disable the button if it doesn't have an effect
   btn_download->Enable((!checkRunning || Core::IsRunning()) && !m_gameid.empty());
 
-  m_listbox_gcodes->Clear();
+  std::sort(m_gcodes.begin(), m_gcodes.end(), helpOrderCodes);  // should sort the codes in alphabetical order  m_listbox_gcodes->Clear();
```

Comment:
Was commenting out the clear intended? If so, it should just be removed. Also comments generally go on top of the line, not off to the side in the codebase.